includes/checkout/functions.php

```diff
@@ -131,8 +131,9 @@ function edd_get_checkout_uri( $args = array() ) {
 		$uri = preg_replace( '/^http:/', 'https:', $uri );
 	}
 
-	if ( edd_get_option( 'no_cache_checkout', false ) && edd_is_caching_plugin_active() )
-		$uri = add_query_arg( 'nocache', 'true', $uri );
+	if ( edd_get_option( 'no_cache_checkout', false ) ) {
```

Comment:
shouldn't the cache busting function do this check? That way extensions could just call it and if its on use it